VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -520,15 +531,21 @@ public void testUnitFragmentCreation() {
         @Parameterized.Parameter(value = 1)
         public Class expected;
 
+        @Parameterized.Parameter(value = 2)
+        public boolean isDiscussionsEnabled;
+
         @Test
-        public void test() {
+        public void test() throws IOException {
+            JsonObject properties = super.generateConfigProperties();
+            properties.addProperty(DISCUSSIONS_ENABLED, isDiscussionsEnabled);
+            config = new org.edx.mobile.util.Config(properties);
```

Comment:
And the `generateConfigProperties()` should also be called before each test is run.